test/unit/definitions/divorce/UserRoleStateAuth.test.js

```diff
@@ -71,36 +65,27 @@ function runTest(authorisationCaseState) {
 }
 
 describe('UserRole authorisations for CaseState', () => {
-  let nonprod = [];
-  let prod = [];
-
-  before(() => {
-    nonprod = getAuthorisationCaseStateDefinitions(
-      [
-        'AuthorisationCaseState',
-        'AuthorisationCaseState-alternative-service-nonprod',
-        'AuthorisationCaseState-alt-service-process-server-nonprod',
-        'AuthorisationCaseState-bailiff-nonprod',
-        'AuthorisationCaseState-deemed-and-dispensed-nonprod',
-        'AuthorisationCaseState-general-referral-nonprod',
-        'AuthorisationCaseState-share-a-case-nonprod',
-        'AuthorisationCaseState-nonprod'
-      ]);
-
-    prod = getAuthorisationCaseStateDefinitions(
-      [
-        'AuthorisationCaseState',
-        'AuthorisationCaseState-prod'
-      ]);
-  });
+  // we need to find out if caa idam role should get R to all states. It seems to be required
 
-  context('should allow minimum R access for all Case States per User Role ', () => {
-    it('(non-prod)', () => {
-      runTest(nonprod);
-    });
+  // context('nonprod', () => {
+  //   it('should allow minimum R access for all Case States per User Role ', () => {
+  //     runTest(nonprod.AuthorisationCaseState, nonprod.AuthorisationCaseType, nonprod.State, nonprod.CaseType);
+  //   });
+  // });
 
-    it('(prod)', () => {
-      runTest(prod);
+  context('prod', () => {
+    it('should allow minimum R access for all Case States per User Role ', () => {
+      runTest(prod.AuthorisationCaseState, prod.AuthorisationCaseType, prod.State, prod.CaseType);
     });
   });
+
+  // context('should allow minimum R access for all Case States per User Role ', () => {
+  //   it('(non-prod)', () => {
+  //     runTest(authorisationCaseStateNonProd);
+  //   });
+  //
+  //   it('(prod)', () => {
+  //     runTest(authorisationCaseStateProd);
+  //   });
+  // });
```

Comment:
Please remove

Reply:
no. Please approve

Reply:
I wanted to leave it here to make sure this will be addressed and fixed. We didn't have this test previously. I mean - we "had", but it didn't actually test what should be tested and now there is quite a few issues with caa role.

I will remove it, but we should come back